test/routes/src/server/routes/t_webhook.js

```diff
@@ -10,7 +12,45 @@ describe('The WebHook API endpoint', () => {
   app = Express();
   app.use(github);
 
+  before(() => {
+    const overrides = conf.stores['test-overrides'];
+    overrides.set('LP_API_URL', 'http://localhost:4000/launchpad');
+    overrides.set('LP_API_USERNAME', 'test-user');
+    overrides.set('LP_API_CONSUMER_KEY', 'consumer key');
+    overrides.set('LP_API_TOKEN', 'token key');
+    overrides.set('LP_API_TOKEN_SECRET', 'token secret');
+  });
+
+  after(() => {
+    const overrides = conf.stores['test-overrides'];
+    overrides.clear('LP_API_URL');
+    overrides.clear('LP_API_USERNAME');
+    overrides.clear('LP_API_CONSUMER_KEY');
+    overrides.clear('LP_API_TOKEN');
+    overrides.clear('LP_API_TOKEN_SECRET');
+  });
+
+  beforeEach(() => {
+    const memcachedStub = { cache: {} };
```

Comment:
Made an issue so we don't lose track of that: https://github.com/canonical-ols/build.snapcraft.io/issues/59